libs/SalesforceSDK/src/com/salesforce/androidsdk/analytics/SalesforceAnalyticsManager.java

```diff
@@ -182,13 +184,77 @@ public static synchronized void reset(UserAccount account, String communityId) {
     }
 
     /**
-     * Sets the publish frequency, in hours.
+     * Sets the interval for periodic background publishing in hours.
+     *
+     * @param periodicBackgroundPublishingHoursInterval The interval for
+     *                                                  periodic background
+     *                                                  publishing in hours
+     * @see #setPublishPeriodicallyOnFrequencyEnabled(boolean)
+     */
+    public static synchronized void setPublishPeriodicallyFrequencyHours(
+            int periodicBackgroundPublishingHoursInterval
+    ) {
+        SalesforceAnalyticsManager.publishPeriodicallyFrequencyHours = periodicBackgroundPublishingHoursInterval;
+        setPublishPeriodicallyOnFrequencyEnabled(true);
+    }
+
+    /**
+     * Sets the interval for periodic background publishing in hours.
+     *
+     * @deprecated Planned for removal 13.0.
+     * Use {@link #setPublishPeriodicallyFrequencyHours(int)} )}.
+     */
+    @Deprecated()
```

Comment:
Without refactoring to make the names consistent, I though the new code wouldn't be very intuitive.  So, here are some deprecations that I believe will make the implementation more friendly while keeping compatibility with 12.0.